web/html/src/manager/errors/default-error-500.js

```diff
@@ -0,0 +1,13 @@
+import React from "react";
+
+const DefaultError500 = () =>
+  <body>
+    <h1>
+      Internal Server Error
+    </h1>
+    <p>
+      Please reload the page.
```

Comment:
To avoid confusions I will change this component name to DefaultJsError

This page is a bit different than the 500.jsp one, it doesn't refer to a server internal error.

Basically, prior React 16 the default behavior of React whenever there is a js error while executing the render is to show a blank page. This behavior used to be different on the react versions < 16, it would keep showing stuff on the UI even if the state is inconsistent.
 
In the new React 16 version, a new feature "errorBoundaries" was introduced in order to have a more reliable error handling. With that feature, we can create some sort of try catchs throughout our components tree and react to error in different ways depending where the error occurs, for instance if an error happens in a deep nested panel we can catch it there and try to recover smoothly instead of disabling the whole page.

This component is the default page if any of those errors isn't catched. This is being defined here https://github.com/lneves12/uyuni/blob/2ca9e651ad68f69326ef66cd24f504c1c3875684/web/html/src/components/general/with-page-wrapper.js#L15